modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss

```diff
@@ -63,6 +63,14 @@ $tab-header-height: 38px;
 	margin: 0;
 	padding: 0.2rem;
 
+	@include media-breakpoint-down(xs) {
+		@at-root {
+			.mobile #{&} {
+				padding-bottom: 7rem;
```

Comment:
I don't like the hardcoded value 7rem but, after a brief with @marcoscv-work, I understand that:
- it's a exclusive fix for mobile
- it's the value of the header that hardly will change
So I think is correct 😁

I have a question: do we intend to change this part with the drilldown-menu?

Reply:
Probably, but not for 7.3 to avoid last-minute instability

Reply:
We will remove all of this is finally use drilldown